docs/src/user-guide/guides-using-presto.md

```
@@ -0,0 +1,178 @@
+    ```
+
+2. Navigate to the `tools/deployment/presto-clp` directory in your terminal.
+3. Run the following script to generate the necessary config for Presto to work with CLP:
+
+    ```bash
+    scripts/set-up-config.sh <clp-json-dir>
+    ```
+
+    * `<clp-json-dir>` is the location of the clp-json package you set up in the previous section.
+
+    Note that for the metadata filter config (i.e., 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)